src/components/dapp-staking/dapp/ProjectDetails.vue

```diff
@@ -101,10 +101,22 @@
         </div>
         <div class="row--social-icons">
           <div v-for="(it, index) in communities" :key="index">
-            <button v-if="it.type === 'Twitter'" class="box--share btn--primary">
+            <button v-if="it.type === CommunityType.Discord" class="box--share btn--primary">
```

Comment:
This row should be aligned as one line
![image](https://user-images.githubusercontent.com/92044428/201336572-ed7d337a-639e-40a6-8b1d-bbcb4971c576.png)